app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,2 @@
+class TeamsController < ApplicationController
```

Comment:
This is more of a habit. Whenever I create a new model, I tend to always add the controller as well. I never really thought about it, but at this stage in the app, it's just unnecessary code. 

I'll break this habit.